composer.json

```diff
@@ -22,12 +22,12 @@
         "issues": "http://github.com/liip/RMT/issues"
     },
     "require": {
+        "ext-json": "*",
```

Comment:
@Jonathanm10 Why do we need this `ext-json`?

Reply:
It's a way to list the PHP extensions that the project requires. It's clearly not mandatory.

Reply:
Isn't it native?

Reply:
Ok, make sense: https://luis-barros-nobrega.medium.com/ext-json-is-missing-in-composer-json-what-is-this-700a1fcad966